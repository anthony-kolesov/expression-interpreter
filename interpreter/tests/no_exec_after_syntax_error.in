# Check that execution error will not happen if there was a syntax erro before
# it.
out 5@2
out ABC
