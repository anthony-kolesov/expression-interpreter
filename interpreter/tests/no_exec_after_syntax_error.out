ERROR:3,5:Unknown symbol `@'
ERROR:3,5:syntax error, unexpected $undefined, expecting $end
